File: src/shared/model/Player.java
package shared.model;

public class Player {
    private final String id;
    private final String name;
    private Role role;
    private boolean isAlive = true;
    private boolean isConnected = true;

    public Player(String id, String name) {
        this.id = id;
        this.name = name;
    }

    public String getId() { return id; }
    public String getName() { return name; }
    public Role getRole() { return role; }
    public void setRole(Role role) { this.role = role; }
    public boolean isAlive() { return isAlive; }
    public void setAlive(boolean alive) { this.isAlive = alive; }
    public boolean isConnected() { return isConnected; }
    public void setConnected(boolean connected) { this.isConnected = connected; }
}

File: src/shared/model/Player.cs
namespace WerewolfGame.Shared.Model
{
    public class Player
    {
        public string Id { get; }
        public string Name { get; }
        public Role Role { get; set; }
        public bool IsAlive { get; set; } = true;
        public bool IsConnected { get; set; } = true;

        public Player(string id, string name)
        {
            Id = id;
            Name = name;
        }
    }
}